### ISSUE TITLE
Jaws doesn't read Preferences Overview

### ISSUE THREAD

**Issue by marlitas** [phetsims/joist#1032]
@KatieWoe noticed this in the pH Scale RC test: https://github.com/phetsims/ph-scale/issues/351

She also saw that this is happening in Greenhouse Effect on main, but not published. I'm not sure if this should be blocking publication right now... @KatieWoe can you confirm that other sims currently in RC aren't seeing this problem either?

**Comment by KatieWoe** [comment]
Just checked membrane transport rc.2 and it also has this problem.

**Comment by jessegreenberg** [comment]
I believe this is a bug that was introduced when we added `accessibleHeading`. The panels with an `accessibleHeading` are still focusable. The "Overview" tab does not have a heading, and therefore does not get a focusable element by default.

Specifically, introduced here: 758c12fe1d82af62ffbe4e91e6f9112926381cbf
